apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/LoggingSnippets.cs

```diff
@@ -211,6 +264,43 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
         // End sample
     }
 
+    /// <summary>
+    /// A simple web application to test the <see cref="GoogleLogger"/>
+    /// and associated classes. Configure <see cref="GoogleLogger"/> to
+    /// write the URL where logs can be found.
+    /// </summary>
+    internal class LoggingTestApplicationWriteUrl
+    {
+        private static readonly string ProjectId = TestEnvironment.GetTestProjectId();
+        internal static readonly TextWriter textWriter = new StringWriter();
+
+        // Sample: RegisterGoogleLoggerWriteUrl
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
+        }
+
+        public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
+        {
+            // Once created, the GoogleLogger will write the URL where logs can be found to the given
+            // textWriter.
+            LoggerOptions loggerOptions = LoggerOptions.Create(loggerDiagnosticsOutput: textWriter);
```

Comment:
If this is a snippet, I'd probably just use `Console.Out` here instead. In many cases that will be absolutely fine. It's harder to test, but I'd be tempted to ignore that for now.